components/logged_in.jsx

```diff
@@ -6,7 +6,6 @@ import $ from 'jquery';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import {loadEmoji} from 'actions/emoji_actions.jsx';
```

Comment:
Do we still need the webapp's loadEmoji action if we're relying on the redux store to do it?

Reply:
It's still used in the emoji_list component, haven't looked into removing it from there yet

Reply:
It can be removed but I'm going to do that in a later PR because it also needs to fetch all the profiles for each emoji and dealing with the fallout of that is something I'm going to handle in a future PR when I add paging to this page